src/interfaces/records/records-interface.ts

```diff
@@ -17,9 +14,7 @@ export async function deleteAllOlderMessagesButKeepInitialWrite(
   tenant: string,
   existingMessages: TimestampedMessage[],
   comparedToMessage: TimestampedMessage,
-  messageStore: MessageStore,
-  dataStore: DataStore,
-  eventLog: EventLog
+  storageController: StorageController,
```

Comment:
i dont think we normally have a trailing `,` after the last parameter

Reply:
Addressed in 7bc13ef